zml/aio/torch.zig

```diff
@@ -314,79 +314,76 @@ pub const PickleData = struct {
             .pers_id => |pers_id| try self.parseValue(allocator, store, prefix, pers_id.ref),
             .seq => |*seq| switch (seq[0]) {
```

Comment:
why are you capture `seq` by reference here ? also I'd rather see `seq` to be destructured into it's type and values, instead of using `seq[0]` and `seq[1]`